Packs/CommonScripts/ReleaseNotes/1_3_27.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### GetDuplicatesMlv2
+- Maintenance and stability enhancements
```

Comment:
Add meaningful release notes.
In this case I think we something important which is this script is going to run on separate docker container each time it called.